tests/integration/dbclient/jdbc/src/test/java/io/helidon/tests/integration/dbclient/jdbc/init/CheckIT.java

```diff
@@ -71,10 +76,15 @@ public void accept(Config config) {
                     connected = true;
                     return;
                 } catch (SQLException ex) {
-                    LOGGER.info(() -> String.format("Connection check: %s", ex.getMessage()));
+                    LOGGER.fine(() -> String.format("Connection check: %s", ex.getMessage()));
```

Comment:
This is not very useful - please add `ex` as parameter to log statement, so we get a stack trace when running in fine or finest log level

Reply:
Done.